doc/develop/west/alias.rst

```diff
@@ -0,0 +1,56 @@
+.. _west-aliases:
+
+West aliases
+############
+
+West allows to add alias commands to the local, global or system configuration files.
+These aliases make it easy to add shortcuts for frequently used, or hard to memorize
+commands for ease of development.
+
+Similar to how ``git`` aliases work, the alias command is replaced with the alias'
+full text and parsed as a new shell argument list. This enables adding argument
+parameters as they were passed to the original command. Spaces are considered
+argument separators, use proper escaping if arguments shouldn't be split.
```

Comment:
Is `shlex` being used here just like for https://docs.zephyrproject.org/latest/develop/west/build-flash-debug.html#permanent-cmake-arguments and https://docs.zephyrproject.org/latest/develop/west/sign.html#rimage ? If yes then do mention `shlex` to clear all parsing and behavior ambiguities